achannarasappa/ticker
# Sorting Quotes

- What I did
Added an option to sort the quotes by `Symbol` or `ChangePercent`.
The quotes are sorted according to the `sort-quotes-by` argument (from configuration file or the cli arguments).
Default sort is by `ChangePercent`.
It is a suggestion to expand to functionality of Sorting. according to issue #5 

- How I did it
Changed `sortQuotes` function in `watchlist.go` and "appended" the relevant order by functionality according to `sort-quotes-by` argument.

- How to verify it
In the configuration file (`ticker.yaml`) or as an argument in the CLI, Set `Symbol` as the value of the flag `sort-quotes-by` in order to verify that the quotes are ordered by symbol name.
Set `ChangePercent` (or nothing) as the value of the flag `sort-quotes-by` in order to order the quotes by the change percentage.
Any other value for the flag, will cause to "ignore" any `orderBy`
